CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# PureScript Community Code of Conduct
+
+The PureScript community follows some simple ground rules to protect safe, inclusive spaces to discuss the PureScript programming language.
+
+All community members are expected to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## What's our purpose?
+
+The PureScript community channels are places to discuss the PureScript programming language. 
+
+* We talk about the language, its related libraries and infrastructure, and our experiences using it.
+* We help PureScript programmers learn the language, and we ask for help becoming stronger programmers ourselves. 
+* We discuss general functional programming concepts and other parts of programming, type theory, and mathematics that are relevant to Purescript.
+* We organize open-source work, meetups, hackathons, and other community events.
+* We discuss what we can do to grow an inclusive, friendly, respectful community.
```

Comment:
Kinda trivial but all your other lists end with an "and". So perhaps this should be "We discuss what we can do to grow an inclusive, friendly, and respectful community."

Reply:
I think this worth adding. It does make things a bit more readable.